packages/create-fuels/src/lib/setupProgram.ts

```diff
@@ -16,12 +16,9 @@ export const setupProgram = () => {
   const program = new Command(packageJson.name)
     .version(packageJson.version)
     .arguments('[projectDirectory]')
-    .option('-c, --contract', 'Include contract program')
-    .option('-p, --predicate', 'Include predicate program')
-    .option('-s, --script', 'Include script program')
-    .option('--pnpm', 'Use pnpm as the package manager')
-    .option('--npm', 'Use npm as the package manager')
-    .option('--bun', 'Use bun as the package manager')
+    .option('--pnpm', 'Use pnpm to install dependencies')
+    .option('--npm', 'Use npm to install dependencies')
+    .option('--bun', 'Use bun to install dependencies')
```

Comment:
Should this also be removed and follow the package manager in use? (as part of https://github.com/FuelLabs/fuels-ts/issues/2459)

There seem to be a bunch of tests and mentions to the `package manager`; it may be worth double-checking everything and removing what's unnecessary.

```suggestion
```

Reply:
In case of duplicates, I'd favor closing the last added issue, not the formers.

I updated the related epic accordingly:
 - https://github.com/FuelLabs/fuels-ts/issues/2468